embrace-android-sdk/embrace-proguard.cfg

```diff
@@ -1,20 +1,33 @@
--keepattributes Exceptions, InnerClasses, Signature, LineNumberTable, SourceFile
```

Comment:
What's the effect of removing the rule to keep these attributes? Instinctively I'd have thought we would want to retain information about inner classes/exception metadata

Reply:
We only need LineNumberTable and SourceFile.
- `Exceptions` keeps info on what exceptions a method might throw (the `throws` part of a function signature). We don't use it.
- `Signature` keeps generics type information. If not present, instead of <T> it'll use Object. We don't use it.
- `InnerClasses` keeps the relationship between classes. If it's not present, inner classes might be renamed and their relationship with their outer class might be lost. This doesn't affects us per se, but some stacktraces might change. Keeping it wouldn't be a problem, but I don't think it's necessary.